backend/api.py

```diff
@@ -3,18 +3,19 @@
 import click
 from flask import Flask
 from flask.testing import FlaskClient
-
 from .config import get_config_from_env
 from .database import db
 from .database import db_cli
+from .auth import user_manager, jwt
 from .routes.incidents import bp as incidents_bp
-# from .routes.auth import bp as auth_bp
+from .routes.auth import bp as auth_bp
 from .utils import dev_only
 
 
 def create_app(config: Optional[str] = None):
     """Create the API application."""
     app = Flask(__name__)
+    app.config["JWT_SECRET_KEY"] = "super-secret"
```

Comment:
Don't add this here! Add this to the `Config` object inside `config.py`, and set it like this. We don't want something like `key = "super-secret"` to ever touch our code, just in case we ever forget to swap it later.

```python
# ...
class Config(object):
    # ...
    JWT_SECRET_KEY = os.environ.get("JWT_SECRET_KEY", os.urandom(64))

Reply:
Moved it, I kept the default as a `secret-key` instead of `urandom` so tokens could persist across local server restarts, is that ok?

Reply:
That's okay for development but we shouldn't allow it in other environments. Maybe set the default to "secret-key" in development and urandom in other envs